sdk/tests/conformance2/rendering/element-index-uint.html

```diff
@@ -265,7 +264,7 @@
     shouldBeUndefined('gl.drawElements(gl.TRIANGLES, 3, gl.UNSIGNED_INT, 0)');
     wtu.glErrorShouldBe(gl, gl.NO_ERROR);
     debug("Enable normals, out-of-range");
-    gl.vertexAttribPointer(normalLoc, 3, gl.FLOAT, false, 7 * sizeInBytes(gl.FLOAT), 4 * sizeInBytes(gl.FLOAT));
+    gl.vertexAttribPointer(normalLoc, 3, gl.FLOAT, false, 7 * sizeInBytes(gl.FLOAT), 21 * sizeInBytes(gl.FLOAT));
```

Comment:
Why is this change being made? It breaks the purpose of the original test, which is that the last fetched normal will be out-of-range. I think this should be reverted, and the test should essentially be made optional.

Microsoft contributed the test sdk/tests/conformance/rendering/out-of-bounds-index-buffers.html , which covers both the situation where the browser does index validation, and where it delegates index validation to the driver.

Could you consider looking at that other test and seeing whether it can be adapted to uint element indices?